spec/client_spec.rb

```diff
@@ -5,7 +5,8 @@
   let(:client) { LocalBitcoins::Client.new(
     client_id: 'CLIENT_ID',
     client_secret: 'CLIENT_SECRET',
-    oauth_token: 'ACCESS_TOKEN'
+    oauth_token: 'ACCESS_TOKEN',
+    use_hmac: ENV['USE_HMAC'],
```

Comment:
I really couldn't think of a clever way to have two sets of tests (one for oauth and one for hmac) without duplicating a lot of code. This way we can run the tests twice like so: `rspec` and `env USE_HMAC=true rspec`. I'm not a huge fan of this so let me know if you think of anything else.